app/controllers/returns/PlasticExportedByAnotherBusinessController.scala

```diff
@@ -69,7 +69,7 @@ class PlasticExportedByAnotherBusinessController @Inject()(
             request.userAnswers
               .setOrFail(AnotherBusinessExportedPage, value)
               .save(cacheConnector.saveUserAnswerFunc(pptId))
-              .map(updatedAnswer => Redirect(returnsNavigator.exportedByAnotherBusinessRoute(updatedAnswer, mode)))
+              .map(updatedAnswer => Redirect(returnsNavigator.exportedByAnotherBusinessRoute(value, mode)))
```

Comment:
could use the "_" sign here

.setOrFail(AnotherBusinessExportedPage, value)
              .save(cacheConnector.saveUserAnswerFunc(pptId))
              .map(updatedAnswer => Redirect(returnsNavigator.exportedByAnotherBusinessRoute(updatedAnswer, mode)))
              .map(_ => Redirect(returnsNavigator.exportedByAnotherBusinessRoute(value, mode)))

Reply:
```suggestion
              .map(_ => Redirect(returnsNavigator.exportedByAnotherBusinessRoute(value, mode)))
```